config/crd/serving.kserve.io_clusterservingruntimes.yaml

```diff
@@ -1,9 +1,11 @@
+
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.12.0
+    controller-gen.kubebuilder.io/version: v0.4.0
```

Comment:
Hey @rachitchauhan43, I noticed you are using old controller-gen version.  The controller-gen version is upgraded to 0.12.0. You should delete the stale binary file inside the bin folder.

Reply:
Thanks for pointing out that. Fixed it.